source/blog/2017-12-04-lvm-configuration-the-easy-way.html.md

```diff
@@ -0,0 +1,107 @@
+---
+title: LVM Configuration the Easy Way
+author: jmarks, nsofer
+tags: LVM, VDSM
+date: 2017-12-04 09:00:00 CET
+---
+
```

Comment:
We need a title:

# Configure LVM filter - the easy way

Reply:
The title is added in the meta tags and is displayed - that's how it works with the blog. See attached screenshot from middleman
![blogpost_lvm_middleman_screenshot](https://user-images.githubusercontent.com/25613123/33604405-8b77c07e-d9be-11e7-9b81-1fcf1b3928ce.png)

Reply:
OK, so we are fine.